app/frontend/src/views/user/Root.vue

```diff
@@ -2,15 +2,15 @@
   <BaseSecure>
```

Comment:
Is this view still used? I can't figure out how to navigate to it, and if I go to it directly I get a 401.

Reply:
Not sure